mimic/model/clb_objects.py

```diff
@@ -212,6 +215,9 @@ class RegionalCLBCollection(object):
                          validator=attr.validators.instance_of(int))
     lbs = attr.ib(default=attr.Factory(dict))
     meta = attr.ib(default=attr.Factory(dict))
+    # Current default limit for batch delete is 10 as per
+    # https://developer.rackspace.com/docs/cloud-load-balancers/v1/api-reference/nodes/#bulk-delete-nodes
```

Comment:
Aah. Here we go.  This link should be in a docstring rather than a comment, ideally; perhaps an `:ivar:` on the class?